File: javascript/1903.Largest-Odd-Number-in-String.js
/**
 * @param {string} num
 * @return {string}
 */
var largestOddNumber = function (num) {
  for (i = num.length - 1; i >= 0; i--) {
    const d = num[i];

    if (["1", "3", "5", "7", "9"].includes(d)) {
      return num.slice(0, i + 1);
    }
  }

  return "";
};

File: python/1903.Largest-Odd-Number-in-String.py
class Solution:
    def largestOddNumber(self, num: str) -> str:
        for i in reversed(range(len(num))):
            d = num[i]

            if d in ['1', '3', '5', '7', '9']:
                return num[0:i+1]
            
        return ''